packages/supabase/src/supabase-client.test.ts

```
@@ -0,0 +1,30 @@
+import { describe, it, expect, beforeEach, vi } from 'vitest';
+import { getSupabaseClient, isSupabaseAvailable } from './supabase-client.js';
+import { createClient } from '@supabase/supabase-js';
+
+vi.mock('@supabase/supabase-js', () => ({ createClient: vi.fn(() => ({})) }));
+
+const env = process.env;
+
+describe('supabase client', () => {
+  beforeEach(() => {
+    process.env = { ...env };
+    delete process.env.PUBLIC_ENV__SUPABASE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)